package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dependency-review-action",
-  "version": "3.0.3",
+  "version": "3.0.4",
```

Comment:
3.0.4 is the current release, this should be 3.0.5.

Reply:
Done - [0574926](https://github.com/actions/dependency-review-action/pull/423/commits/0574926a144976bce4cc4ead75a7733243974ec2)